org.sugarlabs.BaseApp.json

```diff
@@ -174,13 +174,9 @@
             "sources": [
                 {
                     "type": "git",
-                    "url": "https://github.com/TelepathyIM/telepathy-glib.git",
-                    "tag": "telepathy-glib-0.99.11",
-                    "commit": "d00ccc12e7c27b42538389be4595f658ac2cc4eb",
-                    "x-checker-data": {
```

Comment:
we should keep x-checker-data, by updating the pattern to look for 0.24.x

Reply:
So the pattern should be `^telepathy-glib-0.24.\\d$` if I am correct?

Reply:
Test it with https://regex101.com/ and test it with external-data-checker tool (see https://github.com/flathub/org.sugarlabs.BaseApp/tree/branch/23.06#how-to-update)